lorax.spec

```diff
@@ -196,7 +204,9 @@ getent passwd weldr >/dev/null 2>&1 || useradd -r -g weldr -d / -s /sbin/nologin
 %files
 %defattr(-,root,root,-)
 %license COPYING
-%doc AUTHORS docs/livemedia-creator.rst docs/product-images.rst
+%doc AUTHORS
+%doc docs/composer-cli.rst docs/lorax.rst docs/livemedia-creator.rst docs/product-images.rst
+%doc docs/lorax-composer.rst
```

Comment:
Why not `docs/*.rst` ?

Reply:
Because some of the .rst files are useless without sphinx, like index.rst for example.